CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+### Before you Start
+- Unless stated in README.md. Make sure you are pulling the source code from main branch with recursive variable to pull all submodules Collapse need to run.
+
+    ```git pull https://github.com/neon-nyan/Collapse --recursive```
+    
+- Make sure you use the supported IDE/SDKs listed below.
+- Make sure to read the Feature(s) restriction below.
+- When its time to do a Pull Request, do a pull request to the same branch when you first pulled.
+- We don't require a fully descriptive PR description, but please be clear on what added/removed and what it does.

```

Comment:
- Remove `description`, as it is a redundant word (we already use `descriptive`)
- "and what it does" -> and what the code does

Reply:
Applied